moshi/src/main/java/com/squareup/moshi/JsonReader.java

```diff
@@ -528,6 +528,11 @@ private Options(String[] strings, okio.Options doubleQuoteSuffix) {
       this.doubleQuoteSuffix = doubleQuoteSuffix;
     }
 
+    /** Returns a copy of this {@link Options Option's} strings. */
+    public List<String> getStrings() {
+      return Arrays.asList(strings);
```

Comment:
This would allow mutation of items in the backing array.
```suggestion
      return Arrays.asList(strings.clone());
```

Reply:
Oh wow, I actually didn't realize it didn't make a copy of the input array. TIL